addons/point_of_sale/views/res_config_settings_views.xml

```diff
@@ -66,24 +66,34 @@
                             </div>
                         </div>
                     </div>
-                    <h2>Payments</h2>
-                    <div class="row mt16 o_settings_container">
-                        <div class="col-12 col-lg-6 o_setting_box" title="The transactions are processed by Vantiv. Set your Vantiv credentials on the related payment journal.">
+                    <h2>Payment Terminals</h2>
+                    <div class="row mt16 o_settings_container" id="o_settings_payment_terminals">
```

Comment:
use name="o_settings_payment_terminals" instead of id="o_settings_payment_terminals" (don't forget to change reference in enterprise PR)